libbeat/tests/integration/framework.go

```diff
@@ -303,41 +443,116 @@ func EnsureESIsRunning(t *testing.T) {
 
 	esPort := os.Getenv("ES_PORT")
 	if esPort == "" {
-		esPort = "9200"
+		switch scheme {
+		case "http":
+			esPort = "9200"
+		case "https":
+			esPort = "9201"
+		}
```

Comment:
[NIT]
I see it'd break later on if the scheme is neither `http` nor `https`, but it'd be good to have a either a default or to return an error here.

Reply:
added default